docs/source/dev-setup/devenv.rst

```diff
@@ -46,6 +46,125 @@ Prerequisites
 
     brew install jq
 
+Go Programming Language
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Hyperledger Fabric uses the Go Programming Language for many of its
+components.
+
+  - `Go <https://golang.org/dl/>`__ version 1.14.x is required.
+
+There are two environment variables you will need to set properly; you can make these settings permanent by placing them in the appropriate startup file, such as your personal ``~/.bashrc`` file if you are using the ``bash`` shell under Linux.
+
+First, you must set the environment variable ``GOPATH`` to point at the
+Go workspace containing the downloaded Fabric code base, with something like:
+
+.. code:: bash
+  export GOPATH=$HOME/go
+.. note:: You **must** set the GOPATH variable
```

Comment:
This is incorrect. GOPATH is not needed any longer. If you look at the first bit of this [screen cast](https://asciinema.org/a/320081) you will see that the GOPATH is not set. The whole point of modules is to move away from GOPATH.

Reply:
References removed.